lnrpc/neutrinorpc/neutrino.proto

```diff
@@ -244,3 +249,15 @@ message GetBlockHashResponse {
     // The block hash.
     string hash = 1;
 }
+
+message UnbanPeerRequest {
+    // The address of the peer to be unbanned.
+    string peer_address = 1;
+
+    // Indicates if the peer to be unbanned should be connected as a
+    // permanent peer.
+    bool permanent = 2;
```

Comment:
can we make the name more descriptive? maybe something like `perm_connect`? just a suggestion.